### PR TITLE
Refactored the "service-user" lab to test user and group permissions

### PR DESCRIPTION
**Pull request by earfolds**
This lab should now test for WordPress's ability to auto-update, and separately for the presence of a new user with read/write permissions on WordPress's directory.

This should be more tangible for learners, and give them a good starting point for doing their own research into how to solve this problem.

This will require a rethink on how to mark this, and I would also invite discussion on what to rename this lab to, if necessary.

### CODE REVIEW

File: learner-instructions/LO-10-service-permissions.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+So far, when you have deployed WordPress, you have been running it from Apache httpd's default directory with default permissions. This is not ideal, as it means two things:
+
+ - WordPress cannot modify its own files or create new ones.
+ - Server admins need root access to modify WordPress files.
+
+Today, your task has three parts.
+
+1. Host WordPress from the ```/var/www/html/wordpress``` directory so that WordPress has the correct permissions to access its own files.
+2. Harden WordPress by securing the `wp-config.php` file from being read by any other users.
```
Comment (JamesBelchamber) [inline learner-instructions/LO-10-service-permissions.md:11]: Extend this to hint that other files need to be hardened too!

Comment (JamesBelchamber) [reply learner-instructions/LO-10-service-permissions.md:11]: "Harden wordpress following the wordpress hardening guide (link)"

File: learner-instructions/LO-10-service-permissions.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+<details>
+<summary>Medium Hint</summary>
+<br>
+After you've deployed WordPress, you need to modify WordPress file permissions in three ways, to achieve three things. You can do this with three commands.
+</details>
+
+<details>
+<summary>Large Hint</summary>
+<br>
+To harden WordPress, you need to secure wp-config.php from being able to be read by any user other than that which runs the WordPress code.
```
Comment (JamesBelchamber) [inline learner-instructions/LO-10-service-permissions.md:73]: Add a hint that enabling the admin user will deviate from hardening guidelines

Comment (earfolds) [reply learner-instructions/LO-10-service-permissions.md:73]: https://github.com/generation-org/tech-foundations-labs/pull/10/commits/67e6eebf9e8e27c4b392f2f950d5237732a3b70f

File: learner-instructions/LO-10-service-permissions.md (hunk 1 above)
Comment (JamesBelchamber) [reply learner-instructions/LO-10-service-permissions.md:11]: Let's leave this for now and see how learners do with it :)

**Pull request merged**